bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookieWriteLedgerTest.java

```diff
@@ -1191,6 +1209,17 @@ private void readEntries(LedgerHandle lh, List<byte[]> entries) throws Interrupt
         }
     }
 
+    private void readEntries(ReadHandle reader, List<Integer> entries) throws Exception {
+        AtomicInteger index = new AtomicInteger(0);
+        try (LedgerEntries readEntries = reader.read(0, numEntriesToWrite - 1).join()) {
+            readEntries.forEach(e -> {
+                    int origEntry = entries.get(index.getAndIncrement());
+                    int readEntry = e.getEntryBuffer().readInt();
+                    assertEquals(origEntry, readEntry);
```

Comment:
nit: probably adding ledgerid and entryid to the assert message would be helpful in diagnosing in the case of failure

Reply:
Added